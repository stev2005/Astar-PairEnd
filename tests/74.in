ACCTCATTCGGAGAGGGAAGGAAATCAGACTATATAATTCCCCTGTG
15
AGGGGGGCTGTTAGATGTTGGGAAACTGCCG
TATGGGTCAT
ACCTAAGGTCGTCAG
CACGTCCCTTCAAACACGAC
GAGTGCAGTTTTACTCT
AGCCGCGTTCTAACAGCTTCCGCC
CGCTCC
ATAGATCTAA
AGAGTTCACTCGCTAGGGA
GTGATCGTCGTCATTTTA
ATCGCAGTGATAGGGTGTCCA
A
ATATGATGTCAACTAATTAGGGCACGC
CCGCAGCTGACCCGGTCGT
CTGTCTTGCATTCCAGT
