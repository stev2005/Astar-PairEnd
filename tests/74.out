16
4
5
10
8
11
2
3
8
8
9
0
12
10
7
